### ISSUE TITLE
Instructions for others

### ISSUE THREAD

**Issue by flukejones** [a-oboh/epic-games-flutter#1]
Just to make it easier for others, I suggest editing the steps in the readme such that:

1. `git clone https://github.com/a-oboh/epic-games-flutter.git epic_games_ui`
2. `flutter create .`
3. `flutter run -d macos`

This is a lovely little example app.

**Comment by a-oboh** [comment]
Yeah that's cool, not really chanced, you can send in a PR with the changes :)
